public/app/plugins/panel/alertlist/module.tsx

```diff
@@ -148,6 +148,14 @@ const unifiedAlertList = new PanelPlugin<UnifiedAlertListOptions>(UnifiedAlertLi
       },
       category: ['Filter'],
     })
+    .addBooleanSwitch({
+      path: 'showInactiveAlerts',
+      name: 'Show alerts with 0 instances',
+      description:
+        'Include alert rules which have 0 (zero) instances. As these rules have no instances, they will remain hidden if an Alert instance label filter is configured.',
```

Comment:
```suggestion
        'Include alert rules which have 0 (zero) instances. Because these rules have no instances, they remain hidden if the Alert instance label filter is configured.',
```